docs/src/main/docs/guides/15_graalnative.adoc

```diff
@@ -0,0 +1,218 @@
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= GraalVM Native Images
+:description: Helidon SE Native Images
+:keywords: helidon, guide, graalvm
+:helidon-tag: https://github.com/oracle/helidon/tree/{helidon-version}
+:quickstart-example: {helidon-tag}/examples/quickstarts/helidon-quickstart-se
+:docker-multi-stage-url: https://docs.docker.com/develop/develop-images/multistage-build/
+:openjdk-docker-217: https://github.com/docker-library/openjdk/issues/217
+
+This guide describes how to build a GraalVM native image for a Helidon SE application.
+
+== Introduction
+
+https://www.graalvm.org/docs/reference-manual/aot-compilation/[Native images] are ahead-of-time compiled Java code that result in a self
+contained native executable. When used appropriately native images have dramatically faster
+startup and lower runtime memory overhead compared to a Java VM.
+
+In this guide you will learn how to build a native image locally on your machine, as well as using Docker.
+
+== What You Need
+
+|===
+|About 10 minutes
+| <<about/03_prerequisites.adoc,Helidon Prerequisites>>
+| GraalVM CE https://github.com/oracle/graal/releases[19.2.0+ or 19.1.1+]
+|===
+
+== Install GraalVM and the Native Image Command
+
+After https://github.com/oracle/graal/releases[downloading and installing] GraalVM, set the environment:
+
+[source,bash]
+----
+# Your path might be different
+export GRAALVM_HOME=/usr/local/graalvm-ce-19.2.0/Contents/Home/
+----
+
+Then install the optional `native-image` command:
+
+[source,bash]
+----
+$GRAALVM_HOME/bin/gu install native-image
+----
+
+And verify:
+
+[source,bash]
+----
+$GRAALVM_HOME/bin/native-image --version
+----
+
+== Generate The Project
+
+Generate the project using the Helidon SE Quickstart Maven archetype.
+
+[source,bash,subs="attributes+"]
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-se \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-se \
+    -Dpackage=io.helidon.examples.quickstart.se
+----
+
+The archetype generates a Maven project in your current directory
+(for example, `helidon-quickstart-se`). Change into this directory and build.
+
+[source,bash]
+----
+cd helidon-quickstart-se
+----
+
+[source,bash]
+----
+mvn package
```

Comment:
Does make sense combining this box with the one above? If you want to keep it separate I would move "and build it" after the first box.